NEWS.md

```diff
@@ -4,12 +4,22 @@
 
 ## Updates of Existing Functions
 
+* Error messaging throughout the package have been updated from `rlang::abort()` to `cli::cli_abort()`. As a part of the update, each of the `assert_*()` functions have new arguments `assert_*(message, arg_name, call, class).` (#367)
+
+* Warning messaging has also been updated to use {cli} messaging.
```

Comment:
```suggestion
* Warning messaging has also been updated to use ‘{cli}’ messaging.
```

Reply:
That is supposed to backticks , but I’m on my phone

Reply:
updated